tool/tsh/common/proxy.go

```diff
@@ -706,6 +748,19 @@ Use the following command to connect to the database or to the address above usi
   $ {{.command}}
 `))
 
+// dbProxySpannerAuthTpl is the message that's printed for an authenticated spanner db proxy.
+var dbProxySpannerAuthTpl = template.Must(template.New("").Parse(
+	`Started authenticated tunnel for the {{.type}} database "{{.database}}" in cluster "{{.cluster}}" on {{.address}}.
+{{if .randomPort}}To avoid port randomization, you can choose the listening port using the --port flag.
+{{end}}
+` + dbProxyConnectAd + `
+Use the following command to connect to the database or to the address above using other database GUI/CLI clients:
+  $ {{.command}}
+
+Or use a JDBC connection string to connect with other GUI/CLI clients:
+jdbc:cloudspanner://{{.address}}/projects/{{.gcpProject}}/instances/{{.gcpInstance}}/databases/{{.databaseName}}
```

Comment:
could we add the `usePlainText` option here?

Reply:
[1dec6b0](https://github.com/gravitational/teleport/pull/41129/commits/1dec6b00fc0329a370aa4a3bcd1d85648070ed6a)